component/core/src/main/java/org/exoplatform/social/core/jpa/storage/entity/ProfileLabelEntity.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of the Meeds project (https://meeds.io/).
+ *
+ * Copyright (C) 2023 Meeds Association contact@meeds.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.exoplatform.social.core.jpa.storage.entity;
+import org.exoplatform.commons.api.persistence.ExoEntity;
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity(name = "SocProfileLabelEntity")
+@ExoEntity
+@Table(name = "SOC_LABELS ")
+
+@NamedQuery(name = "SocProfileLabelEntity.findLabelByObjectTypeAndObjectIdAndLang", query = "SELECT c FROM SocProfileLabelEntity c WHERE objectType = :objectType and objectId = :objectId and language = :language")
+@NamedQuery(name = "SocProfileLabelEntity.findLabelByObjectTypeAndObjectId", query = "SELECT c FROM SocProfileLabelEntity c WHERE objectType = :objectType and objectId = :objectId")
+
```

Comment:
Make JPA named queries well formatted and easily readable with the following rules (for code maintainability): 
* Use Capital letter for all Keywords
* add each condition in a single line
 
Besides, the `ORDER BY` is missing:
> The use of ORDER BY is mandatory when using query pagination because SQL does not guarantee any particular order unless we provide one via the ORDER BY clause.

```suggestion
@NamedQuery(
  name = "SocProfileLabelEntity.findLabelByObjectTypeAndObjectIdAndLang",
  query = "SELECT c FROM SocProfileLabelEntity c"
      + " WHERE objectType = :objectType"
      + " AND objectId = :objectId"
      + " AND language = :language"
      // ORDER BY for pagiination
)
@NamedQuery(
  name = "SocProfileLabelEntity.findLabelByObjectTypeAndObjectId",
  query = "SELECT c FROM SocProfileLabelEntity c"
      + " WHERE objectType = :objectType"
      + " AND objectId = :objectId"
      // ORDER BY for pagiination
)
```